### PR TITLE
Create 2018-12-20-coreweekly-week-50-2018.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Did you notice the new banner on the Core Weekly? It now matches the new theme of prestashop.com ;-)
+
+Also, last week, a few records have been broken on PrestaShop project's repositories on GitHub, in a week in 2018.
```
Comment (matks) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: `in a week in 2018` looks strange to me 🤔 
I dont know how to say it better though

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+* [#11669](https://github.com/PrestaShop/PrestaShop/pull/11669): Improve error suppression. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11671](https://github.com/PrestaShop/PrestaShop/pull/11671): Use constant instead of function. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11672](https://github.com/PrestaShop/PrestaShop/pull/11672): Non printable character. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11678](https://github.com/PrestaShop/PrestaShop/pull/11678): Fix use of protocol. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11679](https://github.com/PrestaShop/PrestaShop/pull/11679): Fix issue with deleting a category with subcategories on Multistore. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11685](https://github.com/PrestaShop/PrestaShop/pull/11685): php-cs-fixer: Fix "non printable character". Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11689](https://github.com/PrestaShop/PrestaShop/pull/11689): Merge 1.7.5.x in develop, by [@PierreRambaud](https://github.com/PierreRambaud)
```
Comment (matks) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md:60]: Dont know if this one is relevant as it is a "util" Pull Request used to backport fixes on 1.7.5.x branch onto develop

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+
+### Visitors online
+
+* [#2](https://github.com/PrestaShop/statslive/pull/2): getCustomersOnline(): fix SQL clauses to be able to use index. Thank you [@Kioob](https://github.com/Kioob)
+
+
+### GDPR
+
+* [#10](https://github.com/PrestaShop/psgdpr/pull/10): Add export feature + update translations + fix translation issue with csv and more. Thank you [@Joukz](https://github.com/Joukz)
```
Comment (matks) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: Joukz is a prestashop developer

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+### Docker internal images
+
+* [#19](https://github.com/PrestaShop/docker-internal-images/pull/19): Expose HTTPS port, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+## Changes in Documentation
+
+* [#135](https://github.com/PrestaShop/docs/pull/135): Update to Grid docs. Thank you [@sarjon](https://github.com/sarjon)
+* [#168](https://github.com/PrestaShop/docs/pull/168): Add a paragraph in 1.7.5 changes page about the _legacy_link, by [@jolelievre](https://github.com/jolelievre)
```
Comment (matks) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md:189]: Will `_legacy_link` create display issues on the core weekly ? Will markdown consider it styling ?

If yes you can rename it "legacy_link" or "legacy link" ;)

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: yeah, me neither

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md:60]: I usually include it, this is the first time someone notice it ;-)
let's keep it.

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md:189]: I can also escape if this is not working well when merged. Unfortunately, the markdown preview of GitHub is not exactly the same than the Jenkins build.
With jenkins, I usually have issues with |.

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: done

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md (hunk 1 above)
Comment (jolelievre) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: `Also, last week, a few weekly records have been broken on PrestaShop project's repositories on GitHub, for this year 2018.` ?

Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: what do you think of:

> Also, last week, a few 2018 records have been broken on PrestaShop project's repositories on GitHub:

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+Also, last week, a few records have been broken on PrestaShop project's repositories on GitHub, in a week in 2018.
+
+- 78 merged pull request merged in one week
+- 40 pull request merged only for the Core
+- 41 pull requests merged from a single contributor, @MathiasReker who is passionate about code lint 
+- 80% of the PR were from contributors external to the PrestaShop company
+
+It means two things:
+
+ - You are an amazing team of contributors, contributing more and more with quality pull requests, easy to review and to merge
```
Comment (jolelievre) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: `easy to review`.. joker ^^

Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md]: easier instead of easy

**Pull request merged**